src/App.tsx

```diff
@@ -1,5 +1,79 @@
+import styles from "./App.module.css";
+
 function App() {
-  return <div>여기다 작업하세요!</div>;
+  return (
+    <div className={styles.container}>
+      <div className={styles.wrap}>
+        <img src="/class.png" alt="" />
+        <div className={styles.text1}>
```

Comment:
```suggestion
        <div className={styles.textTitle}>
```
이렇게 클래스명에 의미를 담으면 나중에 유지보수가 편합니다...!